### ISSUE TITLE
pedestrian jaywalking

### ISSUE THREAD

**Issue by way-thu** [eclipse-sumo/sumo#15616]
I have a pedestrian who needs to go through the intersection, and the OD is shown below.
![19d5ddef07b074dbd19364b0bfca22b](https://github.com/user-attachments/assets/f69421a1-f912-4188-bcb0-748b327aaace)
I am now simulating his jaywalking behavior using the movetoXY function, and successfully made him walk into the crosswalk when the light was red
![image](https://github.com/user-attachments/assets/37af542b-f035-41ac-8a07-77565450c356)

However, surprisingly, after the control of movetoXY ended, the pedestrian turned around and chose a longer route to reach the destination.
![image](https://github.com/user-attachments/assets/47edb4d5-ea68-4c59-aa81-43154b955af0)

I've tried changing the parameters in the movetoXY method, just like "keeproute", but it doesn't seem to work.

I would like to know how to solve this issue, thank you very much
**SUMO-version:1.18.0

**operating system:**

**Comment by way-thu** [comment]
Theoretically, he should continue to go forward, but he turned around. I don't know what caused this problem
![image](https://github.com/user-attachments/assets/f74ffad5-68c7-41ec-a69e-ecada7df24e7)

**Comment by way-thu** [comment]
https://github.com/user-attachments/assets/29391c00-c2a1-4794-96bd-53e37f5f073a

**Comment by m-kro** [comment]
Yes, that's something we may look into in the future. In the meantime you can still move the pedestrian across the whole crossing using **moveToXY** commands. When the person reaches the other side, it should continue normally to its destination.

**Comment by way-thu** [comment]
Yes, thanks for your advice. I tried to move the person to the other side, it can work well.
